Assets/Base Files (Dont Touch)/Scenes/TestCredits.unity

```diff
@@ -696,6 +696,23 @@ PrefabInstance:
       propertyPath: m_AnchoredPosition.y
       value: 0
       objectReference: {fileID: 0}
+    - target: {fileID: 6715441104139681951, guid: c6119d33da80d8f489c53cee9e12c03c,
```

Comment:
Can you remove the changes to this scene file? Looks like you did this for testing purposes

Reply:
Just reverted it. How does it look now?

Reply:
looks good to me!